test/integration/cli/help.bats

```diff
@@ -5,126 +5,126 @@ load ${BASE_TEST_DIR}/helpers.bash
 @test "cli: show info" {
   run machine
   [ "$status" -eq 0  ]
-  [[ ${lines[0]} =~ "Usage:"  ]]
-  [[ ${lines[1]} =~ "Create and manage machines running Docker"  ]]
+  [[ ${lines[0]} =~ "Usage:"  ]] || false
```

Comment:
@tianon for context, the original issue in bats that I'm trying to work around is https://github.com/sstephenson/bats/issues/49

```
bash-3.2$ set -e
set -e
bash-3.2$ [[ 1 == 2 ]]
[[ 1 == 2 ]] 
bash-3.2$ [[ 1= = 2 ]] || false
[[ 1 == 2 ]] || false
# bash exits
```